src/app/components/views/weather-view/weather-view.component.scss

```diff
@@ -0,0 +1,25 @@
+.card {
```

Comment:
this file has four spaces instead of 2.

also, please add a blank space between each class. and i thought the .rain {} was being removed

Reply:
I discussed having different animations for multiple weather alert types with Micah, we were going to see what it would take to be able to assign different animations to different weather alert types, but if you don't believe it's worth the time I'll just remove it.